src/workerd/api/tests/worker-test.wd-test

```diff
@@ -0,0 +1,19 @@
+using Workerd = import "/workerd/workerd.capnp";
+
+const unitTests :Workerd.Config = (
+  services = [
+    ( name = "worker-test",
+      worker = (
+        modules = [
+          (name = "worker", esModule = embed "worker-test.js"),
+          (name = "module-does-eval", esModule = embed "worker-test-module-does-eval.js"),
+          (name = "another-module-does-eval", esModule = embed "worker-test-another-module-does-eval.js"),
```

Comment:
This can be made simpler by just inlining these modules rather than using embed.. something like:

```suggestion
          (name = "module-does-eval", esModule = "export default eval('1+1')"),
          (name = "another-module-does-eval", esModule = "export default eval('2+2')"),
```

Then removing the two additional fixture files.